client/extensions/woocommerce/app/settings/navigation.js

```diff
@@ -37,6 +37,11 @@ export const SettingsNavigation = ( {
 			path: '/store/settings/taxes/:site',
 			title: translate( 'Taxes' ),
 		},
+		{
```

Comment:
We need some logic added to the navigation component so we can *only* show the email link when the config flag is set to true. Otherwise this link will show up before the feature launches. That would be a check [similar to this](https://github.com/Automattic/wp-calypso/blob/a7fb9077d8fa8fac561bd89a9d247d7752a5f72f/client/extensions/woocommerce/index.js#L161).